chain/indexer/src/handlers.rs

```diff
@@ -722,6 +722,7 @@ where
                     let mut ret = Vec::with_capacity(block_with_logs.logs.len());
 
                     let mut log_tx_hashes = Vec::with_capacity(block_with_logs.logs.len());
+                    let block_id = block_with_logs.to_string();
```

Comment:
This would be better directly in the debugs that are using it.

Reply:
The for loop takes ownership of the block_with_logs.